### PR TITLE
chore: fix skia only solution

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?
Building SkiaOnly Sln falling for wrong target on these two projects

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by Youssef1313 — COMMENTED** [review]
In case it failed CI, you'll need RollForward set to Major

**Review by MartinZikmund — APPROVED** [review]

**Comment by jeromelaban** [comment]
It does fail :) You'll need the RollForward.

File: src/Uno.UI-Skia-only.slnf [hunk 1]
```diff
@@ -33,6 +33,7 @@
       "Uno.Sdk\\Uno.Sdk.csproj",
       "Uno.UI.Adapter.Microsoft.Extensions.Logging\\Uno.UI.Adapter.Microsoft.Extensions.Logging.csproj",
       "Uno.UI.Composition\\Uno.UI.Composition.Skia.csproj",
+      "Uno.UI.Composition\\Uno.UI.Composition.Reference.csproj",
```
Comment (Youssef1313) [inline src/Uno.UI-Skia-only.slnf:36]: This is going to cascade quite badly I think. We may need to consider re-working the packaging for the Graphics3D package.

Comment (ajpinedam) [reply src/Uno.UI-Skia-only.slnf]: @Youssef1313  reverted this change. It seems it was not required.

**Pull request merged**